modules/api/src/main/java/net/countercraft/movecraft/util/Tags.java

```diff
@@ -71,4 +71,16 @@ public static NamespacedKey keyFromString(@NotNull String string){
             return null;
         }
     }
+
+    @NotNull
+    public static EnumSet<Material> getMaterialsFromString(@NotNull String materialName) {
+        EnumSet<Material> returnList = EnumSet.noneOf(Material.class);
```

Comment:
this is a set, not a list (I'm aware this error exists elsewhere in preexisting code, but its better to fix it when writing new code)